src/DynamicRupture/Output/Base.hpp

```diff
@@ -28,16 +28,22 @@ class Base {
     }
 
     bool bothEnabled = generalParams.outputPointType == OutputType::AtPickpointAndElementwise;
-    if (generalParams.outputPointType == OutputType::AtPickpoint || bothEnabled) {
+    bool pointEnabled = generalParams.outputPointType == OutputType::AtPickpoint || bothEnabled;
+    bool elementwiseEnabled = generalParams.outputPointType == OutputType::Elementwise || bothEnabled;
+    if (pointEnabled) {
+      logInfo() << "Enable Dynamic Rupture Receiver Output";
       ppOutputBuilder = std::make_unique<PickPointBuilder>();
       ppOutputBuilder->setMeshReader(&mesher);
       ppOutputBuilder->setParams(reader.getPickPointParams());
-    } else if (generalParams.outputPointType == OutputType::Elementwise || bothEnabled) {
+    }
+    if (elementwiseEnabled) {
+      logInfo() << "Enable Dynamic Rupture 2D Fault Output";
```

Comment:
Enabling 3D fault output?

Reply:
I would have said 2D since the faults are 2D manifolds (neglecting intersections) embedded in 3D space.